python/dgl/nn/pytorch/network_emb.py

```diff
@@ -0,0 +1,220 @@
+"""Network Embedding NN modules"""
+# pylint: disable= invalid-name
+import torch
+from torch import nn
+from torch.nn import init
+from torch.utils.data import DataLoader
+import torch.nn.functional as F
+import numpy as np
+import tqdm
+from ...sampling import random_walk
+
+
+
+class Metapath2vec(nn.Module):
+    """
+    Metapath2vec, a scalable representation learning model for heterogeneous networks
```

Comment:
```suggestion
    metapath2vec module from `metapath2vec: Scalable Representation Learning for
Heterogeneous Networks <https://dl.acm.org/doi/pdf/10.1145/3097983.3098036>`__
```

to be consistent with existing modules

Split it into two lines if it is too long.